### DISCUSSION TITLE
Question about graphics cards

### DISCUSSION THREAD

**Discussion by jamiebill** [ianthehenry/bauble#12] [Q&A]
This could be the wrong place to ask, and it definitely is just a question, and entirely my own "issue". I wanted to ask the version of OpenGL required to run bauble.studio successfully in the browser? I can't get it to run in the browser of my venerable X200. I normally don't do anything graphics-y, and hadn't even looked up what my graphics card is called, but here I am, pining after a functional less-than-15-years-old graphics card. What have you done!

If I'm reading around correctly, my graphics card (Intel GM45) can only run up to version 2.1, which was hip back around 2006 or so. Could you confirm the results of my investigations here, and tell me that this is the reason I can't play around with the lovely 3D graphics? And what version would be the minimum that would work. Thanks!

**Comment by ianthehenry** [comment]
Hi! I'm glad to hear you're interested in Bauble -- sadly Bauble targets OpenGL ES 3.0. It's possible that a browser running ANGLE would be able to emulate it sufficiently to try it out? Although that's kind of a stretch; I don' tknow. Failing that, recent smartphones should be able to run it. Raymarching is relatively expensive, though -- even an older GPU that does support ES 3 might struggle to render Bauble very smoothly.

**Comment by ianthehenry** [comment]
Recent smartphones can run Bauble, although graphics drivers on Android are kind of all over the place.

**Comment by jamiebill** [comment]
Targeting 3.0 already sounds like a generous approach to backwards compatibility! Thanks very much for the clarity, I will look into different ways I can get set up. It does render on my (also very old and crappy) phone.

I also didn't know when I wrote that initial comment that you're the same fellow who wrote the "Janet For Mortals" guide - an amazing piece of work! More power to you.
